packages/components/src/components/input-checkbox/button.css

```diff
@@ -4,14 +4,20 @@
 		grid-template-columns: var(--a11y-min-size) auto;
 		grid-template-areas: 'error error' 'input label' 'hint hint';
 	}
-
-	.button[tabindex]:focus {
+	.button:focus-within {
 		/* override global `[tabindex]:focus` style */
 		cursor: inherit;
+		outline-color: black;
+		outline-style: solid;
 	}
 
 	.button input {
-		display: none;
+		border: none !important;
```

Comment:
Ich vermute das wird an der Stelle so gut funktionieren, ich hätte trotzdem ein paar Anmerkungen:

* Ich fände es gut, klar zu machen was hier passiert - über einen Kommentar oder noch besser indem wir diese Styles in einer utility-Klasse verschieben (`a11y.css`, `.visually-hidden`?)
* Die hier verwendete Technik kenne ich nicht, ich hatte in der Vergangenheit immer [diesen Snippet](https://www.a11yproject.com/posts/how-to-hide-content/) verwendet und weiß, dass er sehr robust ist. Auch unabhängig von dem zu versteckenden Element.
* Brauchen wir die `!important`-Deklarationen wirklich? Wir könnten auch sicher stellen, dass Themes die Styles nicht überschreiben, solche Styles wären dann sowieso redundant.